Source/KSCrash/Recording/KSCrash.m

```diff
@@ -117,6 +117,11 @@ @implementation KSCrash
 
 - (id) init
 {
+    return [self initWithReportFilesDirectory:KSCRASH_ReportFilesDirectory];
```

Comment:
Since this is now a default, let's call it `KSCRASH_DefaultReportFilesDirectory`